specifyweb/frontend/js_src/lib/components/Statistics/Categories.tsx

```diff
@@ -268,14 +265,10 @@ export function Categories({
               ) : null}
               {typeof handleAdd === 'function' ? (
                 <div className="flex gap-2">
-                  <Button.Small
-                    variant={className.secondaryButton}
-                    onClick={(): void => {
-                      setDeletingCategory(categoryIndex);
-                    }}
-                  >
-                    {statsText.deleteCategory()}
-                  </Button.Small>
+                  <DeleteStatsCategory
+                    categoryLabel={label}
+                    onDelete={() => handleRemove?.(categoryIndex, undefined)}
```

Comment:
is this a bug or intentional: you are only displaying the remove button if handleAdd is a function (rather than when handleRemove is a function)?

even if those are coupled at the moment (always both undefined or not), they may not be forever (i.e, imagine a user in the future that does not have delete permissions, but has add permission)